### DISCUSSION TITLE
Générateur de fichiers .json pour frenchATC

### DISCUSSION THREAD

**Discussion by FullGas1** [Nash115/frenchATC-for-MSFS2020#34] [New airport (created by community)]
Bonjour Nash,

tout d'abord merci pour ton programme que j'ai découvert il y a 1 semaine, et qui me semble un complément très intéressant pour la simulation VFR sur MSFS.

Afin de contribuer (modestement) à l'enrichissement de ton travail, j'ai codé un petit générateur de fichier .json qui produit toutes les données pour tous les terrains français. Je te transmets ci-joint un lien vers ce que j'ai produit afin que tu vois si cela mérite d'être intégré à la ta version en ligne sur Github. 
Le zip contient : 
   > le script python du générateur de .json
   > le fichier XML source SIA-DGAC dans lequel je vais chercher les données  
   > le dossier asset généré le 21/5/224

lien drive : [https://drive.google.com/file/d/1_EWkRG4A20_tvsU_uUgYghAFGUCNMvpd/view?usp=drive_link](url)

Je suis à ta disposition pour toute question..
Cdlt

**Comment by FullGas1** [comment]
Bonjour, ci-joint dernière version corrigée de quelques oublis : 
[https://drive.google.com/file/d/1B5nu-HoGQTWDk1AEWIbOpPw5rjsWjM33/view?usp=drive_link](url)

Dans la version précédente j'avais désactivé la génération de "airports-locations.json" pour les tests, et oublié de la remettre avant livraison !

Cdlt

**Comment by FullGas1** [comment]
J'ai trouvé réglé d'autres bug qui faussaient les coordonnées de zone radio de chaque terrain.

Voici la version corrigée : [https://drive.google.com/file/d/1rU6xye8ONC1vpxQL-qNPrLlCfD1Gf2jw/view?usp=drive_link](url)

Tout semble bien se charger, mais j'ai un pb sur mon test. Alors que mon appareil des posé sur LFPZ (Saint Cyr), c'est le terrain LFPN (Toussus le noble) qui apparait dans la fenetre ATC avec les frequences de LFPZ + 1 frequence de LFPV (Villacoublay).

J'avoue que je ne comprend pas pourquoi ?

Par ailleurs, je n'obtiens aucun réponse de LFPZ sur la fréquence sol, alors qu'elle est bien active dans l'appareil.

Si vous avez un moment pour regarder et me dire si cela vient de mes fichiers générés, et ce que je dois faire pour que cela fonctionne ça m'arrangerait.

Par avance merci

Cdlt

**Comment by Nash115** [comment]
Bonjour,
Je regarderai ça pour essayer de comprendre d'où vient le pb...
merci pour le partage et j'espère qu'on comprendra rapidement car cela serait vraiment pratique !
